fieldservice_location_builder/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Field Service Location Builder",
+    "summary": "Adds a tool to help build out Location Hierarchies",
+    "version": "14.0.1.0.0",
+    "category": "Field Service",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/field-service",
+    "depends": ["fieldservice_account_analytic"],
```

Comment:
Why this depends on `fieldservice_account_analytic`? Isn't a very hard dependency for only this tool?